tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -1401,3 +1401,96 @@ def test_timeout_and_cac_card_ejection():
 
     :expectedresults: Satellite should terminate the session after mentioned timeout in setting
     """
+
+
+@tier2
+def test_positive_end_to_end_open_ldap_authsource(
+    test_name, session, ldap_tear_down, open_ldap_data
+):
+    """Create OpenLDAP auth_source with org and loc assigned.
+
+    :id: f2fa0678-ff2f-11ea-b081-d46d6dd3b5b2
+
+    :steps:
+        1. Create a new LDAP Auth source with OpenLDAP, provide organization and
+           location information.
+        2. Fill in all the fields appropriately for OpenLDAP.
+
+    :expectedresults: Whether creating LDAP Auth source with OpenLDAP and
+        associating org and loc is successful.
+    """
+    org = entities.Organization().create()
+    loc = entities.Location().create()
```

Comment:
Don't we have fixtures for that? Can't you use them?

I'm not sure if I follow here. Looks like you use this organization and location when creating ldapauthentication entity, so what would be a problem if you started logged in already in them?